languages/rust/exercises/concept/structs/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+# Instructions
```

Comment:
Could you remove this top-level heading? See [the style guide](https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md).